Snake/BusinessLayer/AI.cs

```diff
@@ -11,40 +11,49 @@ public class AI
         private Direction currentDirection;
         private int X;
         private int Y;
-        Random randomNumber = new Random();
+        Random randomNumber;
         public int XCoordinate
         {
             get
             {
                 return X;
             }
+
+            set

```

Comment:
Grrr still don't like having these public - is it possible to refactor so that the only way to change the AI position is via one of the methods?